RedBeanPHP/Facade.php

```diff
@@ -398,11 +398,49 @@ public static function addDatabase( $key, $dsn, $user = NULL, $pass = NULL, $fro
 			throw new RedException( 'A database has already been specified for this key.' );
 		}
 
+		self::$toolboxes[$key] = self::createToolbox($key, $dsn, $user, $pass, $frozen, $partialBeans);
```

Comment:
This should be
`self::$toolboxes[$key] = self::createToolbox($dsn, $user, $pass, $frozen, $partialBeans);`
You have one too many argument.

Reply:
I changed this in https://github.com/gabordemooij/redbean/pull/815